src/components/Dashboard/TodoList.vue

```diff
@@ -637,8 +626,14 @@ export default class TodoList extends Vue {
     return this.todoItems.sort((a, b) => (a.order - b.order))
   }
 
-  /** Whether the Annual Report verify checkbox should be shown. */
-  protected isShowAnnualReportCheckbox (item: TodoItemIF): boolean {
+  /** Whether to show the invalid section styling. */
+  protected showInvalidSection (item: TodoItemIF): boolean {
+    if (item.isAlteringToBen && !this.isGoodStanding) return true
+    return false
+  }
```

Comment:
I coded this in this manner to more easily support future cases where we'd want to show invalid section.